waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -0,0 +1,47 @@
+{{= {= =} =}}
+import queryCache from '../queryCache'
+
+
+// Map where key is resource name and value is Set
+// containing query ids of all the queries that use
+// that resource.
+const resourceToQueryCacheKeys = new Map()
+
+/**
+ * Remembers that specified query is using specified resources.
+ * If called multiple times for same query, resources are added, not reset.
+ * @param {string} queryCacheKey - Unique key under used to identify query in the cache.
+ * @param {string[]} resources - Names of resources that query is using.
+ */
+export const queryUses = (queryCacheKey, resources) => {
```

Comment:
```suggestion
export const setQueryUses = (queryCacheKey, resources) => {
```
Maybe name it like this or sth similar, so it is clear it is doing some side effect. queryUses sounds also like a function which returns what a specific query uses.

Reply:
Ok, renamed to addResourcesUsedByQuery! And also renamed getQueriesUsing to getQueriesUsingResource.